modules/installation-gcp-tested-machine-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// gcp modules
+
+[id="installation-gcp-tested-machine-types_{context}"]
+= Tested instance types for GCP
+
+The following Microsoft Azure instance types have been *tested* with {product-title}.
```

Comment:
GCP instead?

Reply:
now list the instance family ? @jianli-wei

Reply:
@sadasu  could you help update it ?